examples/arquillian/hollow-jar/README.md

```diff
@@ -0,0 +1,17 @@
+# Hollow JAR / Arquillian testing example
+
+Arquillian is used to deploy a JAX-RS resource and an EJB into the Hollow JAR.
+
+In this scenario, part of the Arquillian test is run as a client 
+(testcase annotated with ```@RunAsClient```) and in-container.
+
+IMPORTANT: ```jmx-remoting``` Galleon layer is required for Arquillian test to work properly.
```

Comment:
We actually shouldn't need this anymore. I've fixed that with https://issues.redhat.com/browse/WFARQ-83.

Reply:
Ah, thank-you, I tested with Beta3 then upgraded to 3.0.0.Final. I just tested and it is no more needed.